test/cfg/runtests.sh

```diff
@@ -100,3 +100,25 @@ else
     fi
 fi
 ${CPPCHECK} ${CPPCHECK_OPT} --inconclusive --library=gtk -f ${DIR}gtk.c
+
+# Check the syntax of the defines in the configuration files
+set +e
+xmlstarlet --version
+XMLSTARLET_RETURNCODE=$?
+set -e
+if [ $XMLSTARLET_RETURNCODE -ne 0 ]; then
+    echo "xmlstarlet needed to extract defines, skipping defines check."
+else
+    for configfile in cfg/*.cfg; do
+        echo "Checking defines in $configfile"
+        # Disable debugging output temorarily since there could be many defines
```

Comment:
Complaning about spelling errors within comments in minor non-production shell scripts would be impertinent...

Reply:
Thanks for the hint :) IMHO there should be no typos too.